docs/UserGuide.md

```diff
@@ -251,8 +256,10 @@ ___
 Adds or deletes genre(s) from a specified anime<br>
 Format: `genre INDEX c/ACTION g/GENRE [g/GENRE] ...`
 
+Notes:
 * Duplicate genres are not allowed
 * You need to provide at least 1 `GENRE`
+* All valid genres provided will be successfully added/deleted from the anime
```

Comment:
phrasing is weird, since valid genres are considered to be those in the genre list. but in this case if there is any genre that isn't in the genre list then the whole command fails
Maybe can specify the same info as line 135
also state that all genres specified will be added/deleted from the anime, if it is not already added/deleted

Reply:
changed